### ISSUE TITLE
security

### ISSUE THREAD

**Issue by ltetrel** [neurolibre/neurolibre-binderhub#9]
Maintain a list of bad users to ban, improve the security of the server..

**Comment by lunebellec** [comment]
For mybinder, it makes sense to exclude domains that are abusing the service. 
But in our case, we should basically block anything, except for the `gitub.com/neurolibre` organization (and possibly others in the future). I am not sure whether such `restrict` rule is already supported.

**Comment by ltetrel** [comment]
https://binderhub.readthedocs.io/en/latest/reference/repoproviders.html#binderhub.repoproviders.RepoProvider

We could have a regex that would match (ban) everything except the ones that contains neurolibre as orga. Something like a negative match.

**Comment by ltetrel** [comment]
We now reject all repos that are not from neurolibre organization

**Comment by lunebellec** [comment]
The test servers should still be open, and the previous approach inspired from mybinder is still useful in that context.

**Comment by ltetrel** [comment]
Agree! For now we still don't have a test binder (for the users) but I can spawn one.

**Comment by ltetrel** [comment]
worth checking : https://github.com/appsecco/attacking-and-auditing-docker-containers-and-kubernetes-clusters
